status-im/status-react
# Backup of contacts through waku

Fixes: #12745 

Adds backup of contacts through waku.


### Testing

1) Create an account
2) Save the seed phrase
3) Add some people to your contacts, or block or add nicknames
4) Make sure that a backup has been performed after you added your contacts (Go to Settings -> Sync settings -> Backup and you can trigger a backup)
5) Uninstall the app
6) Install the app
7) Restore the same seed phrase
8) Check that contacts added in step 3 are there

status: ready
